lib/utp.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (() => {
+  try {
+    const utp = require('utp-native')
+    utp.UTP_SUPPORT = true
+    return utp
+  } catch (err) {
+    console.error('WebTorrent: uTP not supported. Using TCP as fallback')
```

Comment:
We don't want this message to print out in the browser. Can you update the browser exclude information in `package.json` to exclude `./lib/utp.js` instead of `utp-native` since this is now the proper file to exclude in the browser.

Reply:
This error message seems unavoidable, if utp is not supported, shouldn't it be a warning? Or be ignored if specifically passed to webtorrent's options `{utp: false}`?

Reply:
In the browser there's never a way for uTP to be supported. Also, the message says TCP will be used as a fallback, which is also not true. Also, we usually don't use `console.error` to print warnings to the user in WebTorrent.

Once uTP has stabilized a bit more, we should default it to on by default (when the environment supports it) like we do with WebRTC support.